src/en/likemangain/src/eu/kanade/tachiyomi/extension/en/likemangain/LikeMangaIn.kt

```diff
@@ -0,0 +1,14 @@
+package eu.kanade.tachiyomi.extension.en.likemangain
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class LikeMangaIn : Madara(
+    "LikeMangaIn",
+    "https://likemanga.in",
+    "en",
+    dateFormat = SimpleDateFormat("dd MMMM, yyyy", Locale.getDefault()),
```

Comment:
This will change behavior based on the locale used by the user. Changing the locale in Android will not change how the site displays this and will break.

Reply:
Changed to US